scripts/install_base.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 apt-get update -y
-apt-get install -y curl wget
\ No newline at end of file
+apt-get install -y curl wget syslinux
```

Comment:
Docker linking changes the file /etc/hosts and add some env vars. I use gethostip (from syslink) to resolve the ip address for mongo.